x/blob/keeper/params.go

```diff
@@ -21,12 +22,18 @@ func (k Keeper) SetParams(ctx sdk.Context, params types.Params) {
 
 // MinSquareSize returns the MinSquareSize param
 func (k Keeper) MinSquareSize(ctx sdk.Context) (res uint32) {
+	if ctx.BlockHeader().Height < 1 {
+		return appconsts.DefaultMinSquareSize
+	}
```

Comment:
I'm okay with merging as soon as we add a comment or two here, thanks!
```suggestion
	// use the default size for the first block so that we return a value on the
	// first block in PrepareProposal
	if ctx.BlockHeader().Height < 1 {
		return appconsts.DefaultMinSquareSize
	}
```